iceoryx_hoofs/container/include/iox/uninitialized_array.hpp

```diff
@@ -33,11 +33,13 @@ struct ZeroedBuffer
 {
     struct alignas(ElementType) element_t
     {
-        // AXIVION Next Construct AutosarC++19_03-A18.1.1, AutosarC++19_03-M0.1.3 : required by low level UninitializedArray building block and encapsulated in abstraction, declaration of field in struct for usage elsewhere
```

Comment:
Is there no `AutosarC++19_03-A18.1.1` warning anymore?

Reply:
It seems the scan is unreliable :/